### PR TITLE
style: Fix "birthdate", minor consistency edits

### PR DESCRIPTION
**Pull request by mgrhm**
~~This makes the "Edit Contact" screen use the contact's name where appropriate.~~

I've corrected "birthdate" to "birthday" where it appears, and explained why in the specific commit message.

This pull also contains a few stylistic and consistency fixes.

### CODE REVIEW

**Comment by mgrhm** [comment]
I _think_ fixed the conflict.  Hopefully this will be fine to merge.

File: resources/lang/en/people.php [hunk 1]
```diff
@@ -58,11 +58,11 @@
     'people_delete_message' => 'Delete contact',
     'people_delete_confirmation' => 'Are you sure you want to delete :name’s contact? Deletion is immediate and permanent.',
     'people_add_birthday_reminder' => 'Wish happy birthday to :name',
-    'people_add_birthday_reminder_deceased' => 'On this date, :name, would have celebrated their birthday',
+    'people_add_birthday_reminder_deceased' => 'On this date, :name would have celebrated their birthday',
     'people_add_import' => 'Do you want to <a href=":url">import your contacts</a>?',
     'people_edit_email_error' => 'There is already a contact in your account with this email address. Please choose another one.',
     'people_export' => 'Export as vCard',
-    'people_add_reminder_for_birthday' => 'Create an annual reminder for the birthday',
+    'people_add_reminder_for_birthday' => 'Create an annual reminder for :name’s birthday',
```
Comment (asbiin) [inline resources/lang/en/people.php:65]: This message is used when you create a new contact. It's not possible to add the name of the contact.

File: resources/lang/en/people.php [hunk 2]
```diff
@@ -151,14 +151,14 @@
...
-    'information_edit_description' => 'Description (Optional)',
+    'information_edit_lastname' => 'Last name (optional)',
+    'information_edit_description' => 'Description (optional)',
     'information_edit_description_help' => 'Used on the contact list to add some context, if necessary.',
-    'information_edit_unknown' => 'I do not know this person’s age',
-    'information_edit_probably' => 'This person is probably…',
-    'information_edit_not_year' => 'I know the day and month of this person’s birthday, but not the year…',
-    'information_edit_exact' => 'I know the exact birthdate of this person…',
-    'information_edit_birthdate_label' => 'Birthdate',
+    'information_edit_unknown' => 'I do not know :name’s age',
```
Comment (asbiin) [inline resources/lang/en/people.php:157]: This message is used when you create a new contact. It's not possible to add the name of the contact.

File: resources/lang/en/people.php [hunk 3]
```diff
@@ -151,14 +151,14 @@
...
+    'information_edit_probably' => ':name person is probably…',
```
Comment (asbiin) [inline resources/lang/en/people.php:158]: This message is used when you create a new contact. It's not possible to add the name of the contact.

File: resources/lang/en/people.php [hunk 4]
```diff
@@ -151,14 +151,14 @@
...
+    'information_edit_not_year' => 'I know the day and month of :name’s birthday, but not the year…',
```
Comment (asbiin) [inline resources/lang/en/people.php:159]: This message is used when you create a new contact. It's not possible to add the name of the contact.

File: resources/lang/en/people.php [hunk 5]
```diff
@@ -151,14 +151,14 @@
...
+    'information_edit_exact' => 'I know :name’s exact birthday…',
```
Comment (asbiin) [inline resources/lang/en/people.php:160]: This message is used when you create a new contact. It's not possible to add the name of the contact.

File: resources/lang/en/people.php [hunk 6]
```diff
@@ -372,8 +372,8 @@
 
     // Deceased
     'deceased_reminder_title' => 'Anniversary of the death of :name',
-    'deceased_mark_person_deceased' => 'Mark this person as deceased',
-    'deceased_know_date' => 'I know the date this person died',
+    'deceased_mark_person_deceased' => 'Mark :name as deceased',
```
Comment (asbiin) [inline resources/lang/en/people.php:375]: This message is used when you create a new contact. It's not possible to add the name of the contact.

File: resources/lang/en/people.php [hunk 7]
```diff
@@ -372,8 +372,8 @@
...
+    'deceased_know_date' => 'I know the date that :name died',
```
Comment (asbiin) [inline resources/lang/en/people.php:376]: This message is used when you create a new contact. It's not possible to add the name of the contact.

**Review by asbiin — CHANGES_REQUESTED** [review]
Good work! unfortunately, it's not possible to include contact's name in these messages as they are used when creating a new contact

File: resources/lang/en/people.php (hunk 1 above)
Comment (mgrhm) [reply resources/lang/en/people.php:65]: Sorry about that. I’ve removed the changes now.

Comment (asbiin) [reply resources/lang/en/people.php:65]: actually, this one have to be reverted too

File: resources/lang/en/people.php [hunk 8]
```diff
@@ -58,11 +58,11 @@
     'people_delete_message' => 'Delete contact',
     'people_delete_confirmation' => 'Are you sure you want to delete :name’s contact? Deletion is immediate and permanent.',
     'people_add_birthday_reminder' => 'Wish happy birthday to :name',
-    'people_add_birthday_reminder_deceased' => 'On this date, :name, would have celebrated their birthday',
+    'people_add_birthday_reminder_deceased' => 'On this date, they would have celebrated their birthday',
```
Comment (asbiin) [inline resources/lang/en/people.php:61]: The previous change was fine, removing the colon 👍🏽

File: resources/lang/en/people.php (hunk 1 above)
Comment (mgrhm) [reply resources/lang/en/people.php]: Done.

**Review by asbiin — APPROVED** [review]

**Pull request merged**